framework/Models/PostProcessors/PostProcessorFunctions/HS2PS.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
```

Comment:
should this be at the same level of PP? Do we need to still keep the "PostProcessorFunctions" category?

Reply:
It is a folder to encapsulate the post-processors. Let me know if you think it is better to remove it.

Reply:
in followup PR lets remove it, since it might create confusion for new developers (where should I place my PP? etc.)